docs/LOCAL_DEV.md

```diff
@@ -60,6 +60,78 @@ open htmlcov/index.html
 ```
 
 ### To run localization
+
+To enable I18N support, make sure in you settings, following environmental variables are set properly:
+
+```
+USE_I18N = True
+USE_L10N = True
+
+# Django gettext files path:
+# locale/<lang-code>/LC_MESSAGES/django.po, django.mo
+LOCALE_PATHS = (os.path.join(BASE_DIR, "locale"),)
+
+MIDDLEWARE = [
+    ......,
+    "django.middleware.locale.LocaleMiddleware",
+    ......,
+]
+```
+
+Use Chinese as example, run the following command first:
+
 ```
-python3 manage.py makemessages -l en --ignore "venv/*"
+python3 manage.py makemessages -l zh --ignore "extract-strings-env/*"
 ```
+
+The file `locale/zh/LC_MESSAGES/django.po` will be created based on `LOCALE_PATHS` and it looks like:
```

Comment:
@hsong-rh We don't need the contents of the po file, we can just say make changes to the message file djnago.po with the full directory. name